### PR TITLE
Clean unused subscriptions

### PR DESCRIPTION
**Pull request by ikas**
Clean up Subscriptions service, removing code that is not used anymore:
* TerraI alerts
* Forma and Forma250GFW alerts
* Imazon alerts
* Story alerts

### CODE REVIEW

File: app/src/models/layer.js [hunk 1]
```diff
@@ -6,17 +6,6 @@ const LAYERS = [{
...
-    slug: 'imazon-alerts',
-    subscription: true
-}, {
-    name: 'terrailoss',
-    slug: 'terrai-alerts',
-    subscription: true,
-    datasetId: config.get('layers.terraILayerDataset'),
-    layerId: config.get('layers.terraILayer')
 }, {
     name: 'prodes',
```
Comment (edbrett) [inline app/src/models/layer.js]: If there is no cron for it, I think PRODES can go to?

File: app/src/services/imageService.js [hunk 2]
```diff
@@ -10,13 +10,9 @@ const AWS = require('aws-sdk');
 
 const geoQuery = require('services/imageService/geoQuery.json');
 const viirsTemplate = require('services/imageService/template/viirs.json');
-const formaAlertsTemplate = require('services/imageService/template/forma-alerts.json');
-const imazonAlertsTemplate = require('services/imageService/template/imazon-alerts.json');
 
 const LAYERS_PARAMS_MAP = {
     'viirs-active-fires': viirsTemplate,
```
Comment (edbrett) [inline app/src/services/imageService.js]: If there is only one key is this still needed?

File: config/staging.json [hunk 3]
```diff
@@ -19,8 +19,6 @@
     "gladAlertLayer": "8e4a527d-1bcd-4a12-82b0-5a108ffec452",
     "umdLossGainLayerDataset": "b584954c-0d8d-40c6-859c-f3fdf3c2c5df",
     "umdLossGainLayer": "49a80e70-ec52-4ef8-bcc6-fb2771d95b2c",
-    "terraILayerDataset": "68c81bc5-171c-4425-9836-d45659f0ed23",
-    "terraILayer": "50a76478-9f6e-4315-874a-611d10a50338",
     "prodesLayerDataset": "a481466d-de8a-450c-b792-271901d20d3b",
```
Comment (edbrett) [inline config/staging.json]: Same again here for all these PRODES entries. Are they being called?

**Review by edbrett — COMMENTED** [review]
It's so beautiful 🌈 🌈. Great tidy 👏 . Couple of comments but nothing blocking.

File: app/src/models/layer.js (hunk 1 above)
Comment (ikas) [reply app/src/models/layer.js]: Yep :+1:

File: app/src/services/imageService.js (hunk 2 above)
Comment (ikas) [reply app/src/services/imageService.js]: Turns out this whole class (ImageService) is no longer needed with the removal of these alerts, so I removed this service altogether :partying_face:

File: config/staging.json (hunk 3 above)
Comment (ikas) [reply config/staging.json]: Nope, not used, removed as well :+1:

**Pull request merged**